src/freenet/clients/http/PproxyToadlet.java

```diff
@@ -533,7 +533,7 @@ private void showPluginList(ToadletContext ctx, PluginManager pm, HTMLNode conte
 					pluginRow.addChild("td");
 				} else {
 					if (pi.isPproxyPlugin()) {
-						HTMLNode visitForm = pluginRow.addChild("td").addChild("form", new String[] { "method", "action", "target" }, new String[] { "get", pi.getPluginClassName(), "_blank" });
+						HTMLNode visitForm = pluginRow.addChild("td").addChild("form", new String[] { "method", "action", "target", "rel" }, new String[] { "get", pi.getPluginClassName(), "_blank", "noreferrer noopener" });
```

Comment:
It's the second parameter that matters... noreferrer is just there to deal with older browsers